application.py

```diff
@@ -23,4 +23,11 @@
 
 @app.route("/")
 def index():
-    return "Project 1: TODO"
+    return render_template("register.html")
+
+@app.route("/register", methods = ["POST", "GET"])
+def register():
+    name = request.form.get("name")
+    password = request.form.get("password")
+    print(name, "  ", password)
+    return render_template("index.html", name=name)
```

Comment:
check if the method is post or not before getting the form details

- check both name and passwrd are not empty. u can also check for a strong password or not in the frontend
- if registration is successfull render the register.html page with a successfull message or with a failure message

Reply:
Worked as per your feedback Ma'am. Please check